lib/Test/Nginx/Socket.pm

```diff
@@ -2993,6 +3003,14 @@ C<--- raw_request>.
 
 See also the L<TEST_NGINX_USE_HTTP2> system environment for the "http2" test mode.
 
+=head2 curl_options
+
+add curl options when use 'curl' to send request.
+
+=head2 curl_protocol
+
+set protocol such as http/https when use 'curl' to send request.
```

Comment:
`Set protocol (such as http/https) when sending request using 'curl'` may be better here